.github/workflows/mdoc.yml

```diff
@@ -12,6 +12,6 @@ jobs:
           fetch-depth: 0
       - uses: olafurpg/setup-scala@v10
       - uses: olafurpg/setup-gpg@v3
-      - run: sbt docs/docusaurusPublishGhpages
+      - run: sbt '++2.12.13 docs/docusaurusPublishGhpages'
```

Comment:
Ditto, evilplot

Reply:
Ah, I think I got it now.

1. We need 2.13 to be the default version in the build for the `Scala 3 module -> dependsOn Scala 2.13 module` to work
2. `2.12` is needed in docs specifically because it depends on the `plugin` which cannot be built for 2.13 (SBT)

Reply:
Hmm, why does the docs project depend on the sbt plugin? The plugin is enabled in the docs project but there shouldn’t be a project dependency there 🤔

either way, feel free to ignore this comment